packages/playground/src/components/execution/key-value-table.tsx

```diff
@@ -42,13 +42,13 @@ export const KeyValueTable: React.FC<KeyValueTableProps> = ({
     return listItems.map((item, index: number) => {
       if (typeof item.value === 'string') {
         return (
-          <ul className="border-b flex flex-row align-middle" key={index} style={{ paddingLeft: `${level / 2}rem` }}>
-            <li className="pr-2.5 py-2.5 w-1/2 text-studio-7" style={{ paddingLeft: `${level * 1 + 1}rem` }}>
+          <ul className="border-b flex flex-row align-middle" key={index}>
```

Comment:
`index` is not a good key for lists. see https://dev.to/shiv1998/why-not-to-use-index-as-key-in-react-lists-practical-example-3e66